frontend/packages/shared/src/components/GiteaHeader/VersionControlButtons/components/ShareChangesPopover/CommitAndPushContent/FileChangesInfoModal/FileChangesInfoModal.module.css

```
@@ -1,33 +1,11 @@
-.headingWrapper {
-  display: flex;
-  align-items: center;
+.dialog[open] {
+  height: 75vh;
 }
 
-.icon {
-  font-size: 2rem;
-  margin-right: var(--fds-spacing-2);
+.dialog .table {
+  width: 100%;
 }
 
-.fileChangesContainer {
-  display: flex;
-  flex-direction: column;
-  padding: var(--fds-spacing-4) var(--fds-spacing-10) var(--fds-spacing-10) var(--fds-spacing-10);
-  width: 50vw;
-  gap: var(--fds-spacing-3);
-  height: 40vh;
-  justify-content: space-between;
-}
-
-.filePath {
-  padding: var(--fds-spacing-3);
-}
-
-.fileStatus {
-  align-content: flex-start;
-  min-width: 5vw;
-  width: 5vw;
-}
-
-.diffStatus {
-  padding: var(--fds-spacing-3);
+.dialog .table .fileStatusCell {
+  width: 1%;
```

Would this scale better, or am I misunderstanding why it's set to 1%?

```suggestion
  width: min-content;
```

---

I know this is ugly, but it was the best trick I could come up with. `min-content` doesn't work on table cells.

---

Alright, resolved! ☺️